### PR TITLE
Story 21 (JS function) tasks completed

### PR DESCRIPTION
**Pull request by Nadia982**

### CODE REVIEW

File: src/21_function/task4.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const binaryFn = function (a, b) {
+  return a * b;
+};
+
+// 3. Create another function (title might be fn) with 3 parameters: a, b, c where a and b are number type,
+// but c is a reference to the binary function. Function fn calls binary function c passing parameters a, b
+// 4. Call function fn passing numbers 3, 8 and third parameter - previously created binaryFn function. Expected result is 24
+
+const c = binaryFn;
```
Comment (gosimowicz) [inline src/21_function/task4.js:12]: You don't need this const

File: src/21_function/task5.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+//    a. takes an argument and
+//    b. returns a function that returns that argument.
+// E.g. const fnResult = fn(5);
+// fnResult(); // Expected return value is 5
+
+function fn(a) {
+  let fnResult = function () {
+    return a;
+  };
+  return fnResult();
```
Comment (gosimowicz) [inline src/21_function/task5.js:11]: It should return a function so it should be done without ()

**Review by gosimowicz — CHANGES_REQUESTED** [review]

File: src/21_function/task4.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+const binaryFn = function (a, b) {
+  return a * b;
+};
+
+// 3. Create another function (title might be fn) with 3 parameters: a, b, c where a and b are number type,
+// but c is a reference to the binary function. Function fn calls binary function c passing parameters a, b
+// 4. Call function fn passing numbers 3, 8 and third parameter - previously created binaryFn function. Expected result is 24
+
+c = binaryFn;
```
Comment (gosimowicz) [inline src/21_function/task4.js:12]: Heh... The variable c can be removed at all :) that was my goal

**Review by gosimowicz — CHANGES_REQUESTED** [review]

File: src/21_function/task4.js (hunk 3 above)
Comment (Nadia982) [reply src/21_function/task4.js]: updated :)

**Review by gosimowicz — APPROVED** [review]

**Pull request merged**